go/couler/submitter/argo_submitter.go

```diff
@@ -47,8 +47,12 @@ func (submitter *ArgoWorkflowSubmitter) Submit(wf wfv1.Workflow, watch bool) (*w
 			if !ok {
 				continue
 			}
+			fmt.Printf("Statuses of workflow nodes:\n")
+			for _, node := range wf.Status.Nodes {
+				fmt.Printf("Node %s %s. Message: %s\n", node.Name, node.Phase, node.Message)
+			}
```

Comment:
this may introduce lots of messages for big workflows. Why not just print the workflow status?

Reply:
Good catch. I have updated this PR to focus on propagating the workflow error message instead and node statuses are only printed out in tests for debugging purposes.